app/styles/components/c-tooltip-info.scss

```diff
@@ -4,62 +4,62 @@
   border: 0 !important;
   cursor: pointer;
   display: inline-block; // Don't change to inline otherwise the tooltip will be broken
-  position: relative;
+  // position: relative;
 
   .image-icon {
     height: 15px;
     margin-left: 4px;
     width: 15px;
   }
 
-  > .c-tooltip-info-content {
-    background: $color-19;
-    bottom: auto;
-    box-shadow: 0 2px 4px 0 rgba($color-12, .5);
+  .c-tooltip-info-link {
+    border-bottom: 1px solid $color-6;
     color: $color-4;
-    padding: 15px 0;
-    position: absolute;
-    right: 0;
-    text-align: center;
-    top: 0;
-    width: 200px;
-    z-index: 99;
+    font-weight: $font-weight-bold;
+  }
+}
 
-    &::after {
-      border: 6px solid;
-      border-color: transparent transparent $color-19 $color-19;
-      content: '';
-      height: 0;
-      left: 95px;
-      position: absolute;
-      right: 0;
-      top: -4px;
-      transform: scale(1, -1) rotate(-45deg);
-      width: 0;
-    }
+.c-tooltip-info-content {
```

Comment:
Why is this outside `.c-tooltip-info`? I think it should stay inside.

Also, it shouldn't start with `c-` because it's part of the tooltip component. Maybe we should rename `.c-tooltip-info` to `.c-tooltip` so both the JS and the CSS use the same convention. I know it's not something you introduced here but it would make sense.